### PR TITLE
Ai user profile

### PR DESCRIPTION
**Pull request by aizatibraimova**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by!(username: params.fetch(:username))
```
Comment (aborov) [inline app/controllers/users_controller.rb:3]: Great job identifying user by their username!

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <div class="mb-3">
+    <%= form.text_area :body, class: "form-control" %>
   </div>
 
   <div>
-    <%= form.submit %>
+    <%= form.submit class: "btn btn-outline-secondary btn-block" %>
```
Comment (aborov) [inline app/views/comments/_form.html.erb:21]: Nice styling with bootstrap!

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
-
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
```
Comment (aborov) [inline app/views/comments/_form.html.erb:14]: it's nice that you don't make user fill out `photo_id` and their `username`!

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
```
Comment (aborov) [inline app/views/users/show.html.erb:21]: Maybe consider commenting this `img` out for now as it's just a placeholder

**Review by aborov — APPROVED** [review]

File: app/views/users/show.html.erb (hunk 4 above)
Comment (aizatibraimova) [reply app/views/users/show.html.erb:21]: Good catch! I think we will be able to replace it a bit later, so I kept it as it is.

File: config/routes.rb
Comment (jniewiel) [inline config/routes.rb:1]: Nice job with your routes! I'd recommend possibly organizing your routes in alphabetical order in future projects to help with navigating.

File: app/controllers/comments_controller.rb
Comment (jniewiel) [inline app/controllers/comments_controller.rb:1]: Nice job assigning the current_user when creating comments!

**Review by jniewiel — APPROVED** [review]
Great job! Everything looks good.

File: config/routes.rb [hunk 5]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   root "photos#index"
+
+  # get "/users/:id" => "users#show", as: :user
```
Comment (AdolfoNava) [inline config/routes.rb:4]: consider removing the unused lines of code

**Review by AdolfoNava — APPROVED** [review]

File: app/views/comments/_form.html.erb (hunk 3 above)
Comment (JoseMDO) [reply app/views/comments/_form.html.erb:14]: Agreed!

File: app/views/users/show.html.erb (hunk 4 above)
Comment (JoseMDO) [reply app/views/users/show.html.erb:21]: Yes great catch, usually if an image tag is empty, its good either remove it, comment it out, or add default values to it :)

File: config/routes.rb [hunk 6]
```diff
@@ -8,6 +10,7 @@
   resources :comments
   resources :photos
 
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:13]: Nice custom route!

**Review by JoseMDO — APPROVED** [review]
Everything looks good Aizat!

**Pull request merged**